de_workloads/shared_resources/data_factory/adf_linked_services.tf

```diff
@@ -0,0 +1,47 @@
+resource "azurerm_data_factory_linked_service_azure_blob_storage" "ls_Blob_ConfigStore" {
+  name                     = "ls_Blob_ConfigStore"
+  data_factory_id          = data.azurerm_data_factory.factory.id
+  integration_runtime_name = var.integration_runtime_name
+  use_managed_identity     = true
+  service_endpoint         = var.blob_configstore_endpoint
+  storage_storage_kind     = "BlobStorage"
+}
+
+resource "azurerm_data_factory_linked_service_key_vault" "ls_KeyVault" {
+  name            = "ls_KeyVault"
+  data_factory_id = data.azurerm_data_factory.factory.id
+  key_vault_id    = data.azurerm_key_vault.key_vault.id
+}
+
+resource "azurerm_data_factory_linked_service_data_lake_storage_gen2" "ls_ADLS_DataLake" {
+  name                     = "ls_ADLS_DataLake"
+  data_factory_id          = data.azurerm_data_factory.factory.id
+  integration_runtime_name = var.integration_runtime_name
+  use_managed_identity     = true
+  url                      = var.adls_datalake_url
+}
+
+# Make this resource conditional
```

Comment:
Is this comment still needed? It is already made conditional.

Reply:
removed